ui/Decoder.elm

```diff
@@ -13,7 +13,7 @@ buildRecordDecoder =
         |> required "build" buildDecoder
         |> required "download" downloadDecoder
         |> required "source" sourceDecoder
-        |> required "systemaddons" (nullable (list systemAddonDecoder))
+        |> required "systemaddons" (maybe (list systemAddonDecoder))
```

Comment:
ah, it seems it's because the `SystemAddon` decoder isn't correct: It's expecting a `builtin_version` and a `update_version` but there's none of those in the data.

I guess the decoder should instead be:
```elm
systemAddonDecoder : Decoder SystemAddon
systemAddonDecoder =
    decode SystemAddon
        |> required "id" string
        |> optional "builtin" string ""
        |> optional "updated" string ""
```